back/engines/commercial/id_criipto/spec/requests/criipto_verification_spec.rb

```diff
@@ -111,17 +115,46 @@
 
     expect(response).to redirect_to('/en/yipie?random-passthrough-param=somevalue&verification_success=true')
 
-    expect(user.reload).to have_attributes({
-      verified: true
-    })
+    expect(user.reload).to have_attributes(verified: true)
     expect(user.custom_field_values['municipality_code']).to eq '0173'
     expect(user.custom_field_values['birthyear']).to eq 1977
     expect(user.custom_field_values['birthdate']).to eq '1977-12-31'
     expect(user.verifications.first).to have_attributes({
       method_name: 'criipto',
       user_id: user.id,
       active: true,
-      hashed_uid: '203fb09eaa8e93ee8439b92c4ce8a4e47ab820c1b87bd7d9772376cbd1e63529'
+      hashed_uid: Verification::VerificationService.new.send(:hashed_uid, '{29d14ea0-6e16-4732-86ac-5de87a941784}', 'criipto')
```

Comment:
Unless this it is not stable, I think it's always better to put a literal value in the specs than running through the same codepaths. What if we break something in `hashed_uid`?

Reply:
> I trust you this change is needed

Sorry, I just realized that the bug was not explained in this PR, it was only explained in Slack. Just added it to the PR description.

> What if we break something in `hashed_uid`?

I just wanted to make clear what value is being hashed to prevent this bug (hashing `nil` in the future). But your point is very valid. I believe nothing blocks me from checking two values (a literal value and the result of `hashed_uid`) in one spec to cover both cases.